docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,235 @@
+to build the image:
+
+```Dockerfile
+FROM busybox
+ADD _output/local/go/bin/kube-scheduler /usr/local/bin/kube-scheduler
+```
+
+Save the file as `Dockerfile`, build the image and push it to a registry. This example
+pushes the image to
+[Google Container Registry (GCR)](https://cloud.google.com/container-registry/).
+For more details, please read the GCR
+[documentation](https://cloud.google.com/container-registry/docs/).
+
+```shell
+$ docker build -t my-kube-scheduler:1.0 .
+$ gcloud docker push gcr.io/my-gcp-project/my-kube-scheduler:1.0
+```
+
+### 2. Define a Kubernetes Deployment for the scheduler
+
+Now that we have our scheduler in a container image, we can just create a pod
+config for it and run it in our Kubernetes cluster. But instead of creating a pod
+directly in the cluster, let's use a
+[Deployment](http://kubernetes.io/docs/user-guide/deployments/) for this example. A
+[Deployment](http://kubernetes.io/docs/user-guide/deployments/) manages a
+[Replica Set](http://kubernetes.io/docs/user-guide/replicasets/) which in turn manages
+the pods, thereby making the scheduler resilient to failures. Here is the deployment
+config. Save it as `my-scheduler.yaml`:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: my-scheduler
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      containers:
+      - command: [/usr/local/bin/kube-scheduler, --address=0.0.0.0,
+                  --scheduler-name=my-scheduler]
+        image: gcr.io/my-gcp-project/my-kube-scheduler:1.0
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+          initialDelaySeconds: 15
+        name: kube-second-scheduler
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+        resources:
+          requests:
+            cpu: '0.1'
+        securityContext:
+          privileged: false
+        volumeMounts: []
+      hostNetwork: false
+      hostPID: false
+      volumes: []
+```
+
+An important thing to note here is that the name of the scheduler specified as an
+argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the optional `scheduler.alpha.kubernetes.io/name` annotation on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
+
+Please see the
+[kube-scheduler documentation](http://kubernetes.io/docs/admin/kube-scheduler/) for
+detailed description of other command line arguments.
+
+### 3. Run the second scheduler in the cluster
+
+In order to run your scheduler in a Kubernetes cluster, just create the deployment
+specified in the config above in a Kubernetes cluster:
+
+```shell
+$ kubectl create -f my-scheduler.yaml
```

Comment:
Removed the $ here.

@bgrant0607 Did not split into separate blocks anywhere in this file because there were no quotes or yaml in the output. But feedback taken.